guides/Install-API-instance.md

```diff
@@ -0,0 +1,109 @@
+
+To install Ansible, check [its documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html).
+
+To check that Ansible is well installed, try to run the following command:
+
+```bash
+ansible --version
+
+# It should output something like:
+
+# ansible [core 2.11.2]
+#   config file = /etc/ansible/ansible.cfg
+#   [...]
+```
+
+For information, during the redaction of this guide, we used Ansible version 2.11.2 with Python 3.9.4 (Ansible is written in Python). Note that the Python version used to run Ansible is not related to the [Python version recommended by OpenFisca Core](https://github.com/openfisca/openfisca-core#environment) (which is Python 3.7 at the time we write this guide).
+
+The Ansible _playbook_ is designed to be run on a managed node that runs [Ubuntu 20.04.2.0 LTS (Focal Fossa)](https://releases.ubuntu.com/20.04/), the latest long term support (LTS) version at the time the _playbook_ was written.
+
+## Configuration
+
+The default values of the variables used by the _playbook_ are defined in [`defaults/main.yml`](../ansible/roles/openfisca-api/default/main.yml).
+
+## Install OpenFisca Web API
+
+The following sub-sections describe how to run the `openfisca-api` _playbook_ with 2 scenarios:
+
+- on a remote server for production
+- on a local virtual machine (VM) for development
+
+### Remote server scenario
+
+This scenario describes how to install OpenFisca Web API on a remote server.
+
+There are some prerequisites to running the _playbook_:
+
+- a server must be available, running [Ubuntu 20.04.2.0 LTS (Focal Fossa)](https://releases.ubuntu.com/20.04/)
+- it must expose an SSH connection
+- the _control node_ (the machine that runs the _playbook_) must be able to access the server via SSH as `root`
+- a domain name must have been created, and it must target the public IP address of this server
+
+The _playbook_ refers to the server using `openfisca_api`, a symbolic host name that does not necessarily correspond to a real server host name or domain name (cf `hosts` property in [`openfisca-api.yml`](../ansible/openfisca-api.yml)).
+To associate a concrete IP address or domain name to that symbolic host name, we use an Ansible inventory that can define host groups with a symbolic name (i.e. `openfisca_api` in this case) targeting one or many concrete host names.
+
+Inventories are stored as sub-directories of the [`inventories`](../ansible/inventories/) directory. Each inventory provides at least a `hosts` file, and can also overload _playbook_ default variables by providing a `group_vars/openfisca_api.yml` file.
+
+An SSL certificate is issued from Let's Encrypt if `enable_ssl` and `letsencrypt_email` variables are both defined.
+
+To run the _playbook_, run the following command.
+
+```bash
+ansible-playbook -i ansible/inventories/scaleway/ ansible/openfisca-api.yml
```

Comment:
Do we really need to run the `scaleway` inventory? Should it not be rather the inventory that the user just created? 🙂